### PR TITLE
update to validate 1.26

### PR DESCRIPTION
**Pull request by jcheld**
Signed-off-by: Held, Jarad <jaradheld@gmail.com>

### CODE REVIEW

**Review by arkodg — DISMISSED** [review]
LGTM thanks for taking care of this !

**Comment by jcheld** [comment]
Looking at why conformance test failed

File: .github/workflows/build_and_test.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
     needs: [build]
     strategy:
       matrix:
-        version: [ v1.23.6, v1.24.0, v1.25.3 ]
+        version: [ v1.24.0, v1.25.3, 1.26.0 ]
```
Comment (danehans) [inline .github/workflows/build_and_test.yaml:76]: s/1.26.0/v1.26.0/

**Review by danehans — CHANGES_REQUESTED** [review]

File: .github/workflows/build_and_test.yaml (hunk 1 above)
Comment (arkodg) [reply .github/workflows/build_and_test.yaml:76]: good catch, this is why CI is failing, can't find the kind docker image with the right tag

**Comment by danehans** [comment]
Waiting for https://github.com/envoyproxy/gateway/pull/1021 to merge and will then rerun CI.

**Comment by danehans** [comment]
CI is still running `GatewayObservedGenerationBump` test, @jcheld please rebase to consume https://github.com/envoyproxy/gateway/commit/d016a662876abad67a6481f6c334f05bd15a262b.

**Review by danehans — APPROVED** [review]

**Pull request merged**